node/internal/actors/mock_node_test.go

```diff
@@ -0,0 +1,87 @@
+package actors
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/nats-io/nkeys"
+	"github.com/synadia-io/nex/models"
+	actorproto "github.com/synadia-io/nex/node/internal/actors/pb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+const (
+	NodeXkey    string = "SXAAJQZR4DLEWWUAIDYHIV3JRXNHYMKV5O3POAWXQLATMPAF2OWLJTZ4BM"
+	NodeXkeyPub string = "XC6LBRSGPEQTOFBYEII2DX2RVWXEK27BARNJMX2V3WIKR6XIWFLRYYZY"
```

Comment:
Nit, the `string` declaration is not needed.

Reply:
Oh no! I nit when it isnt there 🤣